frontend/src/concepts/pipelines/content/ManagePipelineServerModal.tsx

```diff
@@ -136,13 +203,38 @@ const ViewPipelineServerModal: React.FC<ViewPipelineServerModalProps> = ({
                     />
                   </DescriptionListDescription>
                 </DescriptionListGroup>
+                <PipelineCachingSection
+                  enableCaching={enableCaching}
+                  setEnableCaching={setEnableCaching}
+                  variant="description"
+                />
               </DescriptionList>
```

Comment:
This is not aligned with the other section
<img width="766" height="399" alt="Screenshot 2025-08-04 at 2 20 13 PM" src="https://github.com/user-attachments/assets/676f8091-30b4-47c6-86a4-e24a9aa3fa8c" />